SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf

```diff
@@ -109,21 +93,25 @@
 	<owl:Class rdf:about="&fibo-sec-dbt-ab-pbs;PrincipalProtectedNote">
 		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ab-pbs;StructuredFinanceInstrument"/>
 		<rdfs:label xml:lang="en">principal protected note</rdfs:label>
-		<skos:definition xml:lang="en">structured finance that offers investors exposure to chosen underlying assets using various approaches and offering a large variety of asymmetric pay-off profiles</skos:definition>
+		<skos:definition xml:lang="en">structured finance that offers investors exposure to chosen underlying assets using various approaches and asymmetric pay-off profiles</skos:definition>
```

Comment:
this doesn't differentiate from any other structured finance instrument nor bring in the aspect of principal protection. The class name seems too general - I'm sure there are other principal protected notes that are not structured finance e.g., inflation rate bonds

Reply:
@rivettp Ok - I'll see if I can find differentiating characteristics to include in the definition, but that may require assistance from JG.